src/webview/wkwebview/mod.rs

```diff
@@ -663,38 +663,13 @@ r#"Object.defineProperty(window, 'ipc', {
       // Inject the web view into the window as main content
       #[cfg(target_os = "macos")]
       {
-        let parent_view_cls = match ClassDecl::new("WryWebViewParent", class!(NSView)) {
-          Some(mut decl) => {
-            decl.add_method(
-              sel!(keyDown:),
-              key_down as extern "C" fn(&mut Object, Sel, id),
-            );
-
-            extern "C" fn key_down(_this: &mut Object, _sel: Sel, _event: id) {
-              // do nothing; this prevents macOS from playing an unsupported key feedback sound
-            }
-
-            decl.register()
-          }
-          None => class!(NSView),
-        };
-
-        // Create a view to contain the webview, without it devtools will try to
-        // inject a subview into the frame causing an obnoxious warning.
-        // See https://github.com/tauri-apps/wry/issues/273
-        let parent_view: id = msg_send![parent_view_cls, alloc];
-        let _: () = msg_send![parent_view, init];
-        parent_view.setAutoresizingMask_(NSViewHeightSizable | NSViewWidthSizable);
```

Comment:
will the removal of this disable auto resizing of the webview based on the parent? the current behavior is that the webview content will automatically fill the whole window when the window is resized.

Reply:
Yes, it will be auto-resized when the window is resized.